pkgs/development/tools/wp-cli/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, writeText, bash, fetchurl, php }:
+
+let
+  phpIni = writeText "wp-cli-php.ini" ''
+    [Phar]
+    phar.readonly = Off
+  '';
+
+in stdenv.mkDerivation rec {
+  version = "0.23.1";
+  name = "wp-cli-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/wp-cli/wp-cli/releases/download/v${version}/${name}.phar";
+    sha256 = "1sjai8gjsx6j82lsxq9m827bczp4ajnldk6ibj4krcisn9pjva5f";
+  };
+
+  propagatedBuildInputs = [ php ];
+
+  buildCommand = ''
+    mkdir -p $out/bin
+
+    cat >$out/bin/wp <<EOF
+    #! ${bash}/bin/bash -e
+    exec ${php}/bin/php -c ${phpIni} -f ${src} \$*
```

Comment:
Maybe you want to use ```"$@"``` instead of ```$*``` here? (With proper escaping so that becomes the end result in the generated file, of course.) I think I would.

http://stackoverflow.com/questions/12314451/accessing-bash-command-line-args-vs